tests/unit/specs/scripts/num.spec.js

```diff
@@ -80,6 +81,22 @@ describe(`number helper`, () => {
     expect(percent(0.2612)).toBe(`26.12%`)
   })
 
+  it(`should format a very big percent. Convert it to trillions`, () => {
+    expect(percentBig(`8719010560462937.9`)).toBe(`8,719.01 T %`)
+  })
+
+  it(`should format a very big percent. Convert it to billions`, () => {
+    expect(percentBig(`719010560462.9379`)).toBe(`719.01 B %`)
+  })
+
+  it(`should format a very big percent. Convert it to millions`, () => {
+    expect(percentBig(`7010560.46`)).toBe(`7.01 M %`)

```

Comment:
should probably just be `7 M`

i think 1 decimal place is customary for giant trunacted numbers like this (for all `bigPercent`)

Reply:
OK, let me change that